op-e2e/system_test.go

```diff
@@ -171,21 +172,21 @@ func TestL2OutputSubmitter(t *testing.T) {
 	// Wait for batch submitter to update L2 output oracle.
 	timeoutCh := time.After(15 * time.Second)
 	for {
-		l2ooTimestamp, err := l2OutputOracle.LatestBlockTimestamp(&bind.CallOpts{})
+		l2ooBlockNumber, err := l2OutputOracle.LatestBlockNumber(&bind.CallOpts{})
 		require.Nil(t, err)
 
 		// Wait for the L2 output oracle to have been changed from the initial
 		// timestamp set in the contract constructor.
-		if l2ooTimestamp.Cmp(initialSroTimestamp) > 0 {
+		if l2ooBlockNumber.Cmp(initialOutputBlockNumber) > 0 {
 			// Retrieve the l2 output committed at this updated timestamp.
-			committedL2Output, err := l2OutputOracle.GetL2Output(&bind.CallOpts{}, l2ooTimestamp)
```

Comment:
@trianglesphere should this check that the return value is non-zero?

Reply:
Probably. There's also the option to revert inside the contract.
Fixing this proper likely means adding a second wrapper around the contract

Reply:
>  a second wrapper around the contract

I assume you mean a second wrapper in Go?
I don't love adding a revert here, as it will make it more complicated to handle in any contract that calls this function, and generally we'd prefer to move complexity off-chain.

Reply:
yea, a wrapper around the go bindings (which is a wrapper around the eth & solidity API).
Let's manually do this check for now & build golang wrapper later.

Reply:
its kind of like the case in erc721 when `ownerOf(uint256)` reverts when the nft hasn't been minted

Reply:
OK, I reconsidered and actually think that this is a good idea and will add in a future PR.
I've created a separate issue to track it.